tasks/quality.js

```diff
@@ -49,8 +52,8 @@ module.exports = function(grunt) {
     var phpStandard = grunt.config('config.phpcs.standard')
       || 'vendor/drupal/coder/coder_sniffer/Drupal';
 
-    var ignoreError = grunt.config('config.phpcs.ignoreExitCode');
-    ignoreError = ignoreError === undefined ? true : ignoreError;
+    var ignoreError = grunt.config('config.validate.ignoreError');
```

Comment:
We should probably preserve the existing config as functioning to avoid the immediate B/C break. Mark as deprecated and possibly output a warning?